src/stylesheets/components/_button-action.scss

```diff
@@ -0,0 +1,16 @@
+// -----------------------------------------------------------------------------
+// This file contains all styles related to the button-empty component.
+// -----------------------------------------------------------------------------
+
+.button-action {
+  text-transform: uppercase;
+  padding: 20px 50px;
+  background-color: #00897b;
```

Comment:
@Dudnik25  Use css variables, for example `var(--brand-clr);`
Check out all color variables in the _variables.scss module.

Also you can use scss function which makes things shorter, for example: `v(brand-clr);`

Apply the same for all colors in your section.